deploy/webhook-v2/deployment.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: admin-serviceaccount
+  namespace: default
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: admin-crb
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+- kind: ServiceAccount
+  namespace: default
+  name: admin-serviceaccount
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: docs-controller
```

Comment:
if this is getting deployed to the same cluster as the v1 docs deployment, I think we'll have the change the name right? Since we'll have multiple resources following the same `namespace:resourcetype/name` pattern?

Reply:
It is deployed to a different cluster (I didn't want to take any chances).  It is deployed to `docs-v2` cluster (vs `docs` cluster)